Directory.Packages.props

```diff
@@ -2,47 +2,69 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <NoWarn>NU1507</NoWarn>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
```

Comment:
This allows us to specify package versions for transitive packages and takes care of promoting them to a top-level dependency when necessary.